frontend/nuxt.config.ts

```diff
@@ -1,116 +1,33 @@
 import { defineNuxtConfig } from "nuxt/config"
 
-import { LOCAL, PRODUCTION } from "./src/constants/deploy-env"
-
+import { disallowedBots } from "./src/constants/disallowed-bots"
 import locales from "./src/locales/scripts/valid-locales.json"
-import { meta as commonMeta } from "./src/constants/meta"
 
 import type { LocaleObject } from "@nuxtjs/i18n"
 
-const favicons = [
-  // SVG favicon
-  {
-    rel: "icon",
-    href: "/favicon.ico",
-  },
-  {
-    rel: "icon",
-    href: "/openverse-logo.svg",
-  },
-  // SVG favicon for Safari
-  {
-    rel: "mask-icon",
-    href: "/opvenverse-logo.svg",
-    color: "#30272E",
-  },
-  // Fallback iPhone Icon
-  {
-    rel: "apple-touch-icon",
-    href: "/openverse-logo-180.png",
-  },
-]
-
-const disallowedBots = [
-  "GPTBot",
-  "CCBot",
-  "ChatGPT-User",
-  "Google-Extended",
-  "anthropic-ai",
-  "Omgilibot",
-  "Omgili",
-  "FacebookBot",
-  "Diffbot",
-  "Bytespider",
-  "ImagesiftBot",
-  "cohere-ai",
-]
-
 /**
- * Robots.txt rules are configured here via the \@nuxtjs/robots package.
- * @see {@link https://nuxtseo.com/robots/guides/nuxt-config|Robots Config Rules}
+ * This configuration sets values necessary at *build time*
+ * to build the app and prerender our pages.
```

Comment:
> In that sense, the "definitive" list of supported variables shouldn't be .env.template, but the runtimeConfig property of this nuxt configuration and whatever is defined in app.config.ts and pulled from import.meta.env.

If the .env.template is the union of the runtimeConfig values in nuxt.config.ts and the values is defined in app.config.ts, isn't it actually a pretty good "definitive" list of supported variables? Perhaps then a comment in the .env.template explaining that the values come from those two locations would be a better replacement for these docs.

Reply:
I suppose so, I just don't see why `.env.template` is necessary, especially because it encourages/requires us to define defaults in two places: that .env.template file _and_ in nux.config.ts's runtimeConfig object.

Reply:
That's a fair point. We probably don't even document it well, and since the frontend doesn't _require_ a `.env` for local development, we might not even need the template. Provided we have some documentation about `ov env` or other mechanisms for setting local runtime variables.

Reply:
I've added the patch, @sarayourfriend.

I also removed the `env.template` because it's not necessary, as is stated in the comments here.

Runtime config in `nuxt.config` should have the values used for the dev environment, if they are different from the default values. These should be overriden by the runtime variables in the infrastructure, specific for each environment (staging/production).

Build time variables are in set in `appConfig`.